### DISCUSSION TITLE
game not loading

### DISCUSSION THREAD

**Discussion by NickyBoi92** [X2CommunityCore/xcom2-launcher#395] [Help]
AML Version: 1.5.1
i downloaded this AML and have gone through all my mods which is said to all be ok but when i launch the game the game is instantly not responding. i dont even get to the loading screen that goes to main menu

**Comment by Dragon-32** [comment]
If AML has launched the game then it's done its job

Your problem sounds like with the mods you're trying to run.
https://old.reddit.com/r/xcom2mods/wiki/mod_troubleshooting
